ee/tabby-db/src/lib.rs

```diff
@@ -287,124 +271,17 @@ where
 
 impl<T> Clone for DbOption<T>
 where
-    T: Clone + DbNullable,
+    T: Clone,
 {
     fn clone(&self) -> Self {
         self.0.clone().into()
     }
 }
 
-// FIXME: migrate all DateTimeUtc usage to reduce complexity
-#[deprecated(
-    note = "use `DateTime<Utc>` instead, which can be specified in query! or query_as! in grammar like `created_at as \"created_at: DateTime<Utc>\"`"
-)]
-#[derive(Default, Clone)]
-pub struct DateTimeUtc(DateTime<Utc>);
-
-impl Display for DateTimeUtc {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.as_sqlite_datetime())
-    }
-}
-
-impl std::fmt::Debug for DateTimeUtc {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.as_sqlite_datetime())
-    }
-}
-
-impl Add<Duration> for DateTimeUtc {
-    type Output = Self;
-
-    fn add(self, rhs: Duration) -> Self::Output {
-        ((self.0) + rhs).into()
-    }
-}
-
-impl Sub<Duration> for DateTimeUtc {
-    type Output = Self;
-
-    fn sub(self, rhs: Duration) -> Self::Output {
-        ((self.0) - rhs).into()
-    }
-}
+pub type DateTimeUtc = DateTime<Utc>;
```

Comment:
do not use type alias, just use `DateTime<Utc>` every where

Reply:
OK, it has been removed.